src/fastgltf.cpp

```diff
@@ -204,90 +204,97 @@ namespace fastgltf {
 		return Error::InvalidJson;
 	}
 
+    template <typename X, typename T, std::size_t N>
+    [[nodiscard, gnu::always_inline]] Error copyNumericalJsonArray(simdjson::ondemand::array& array, std::array<T, N>& target) {
```

Comment:
```suggestion
    [[nodiscard, gnu::always_inline]] inline Error copyNumericalJsonArray(simdjson::ondemand::array& array, std::array<T, N>& target) {
```

Reply:
GCC won't inline the function unless you mark it as inline.

Reply:
Note that I don't know whether it should be inlined, this was just a side remark. I have not benchmarked.